lib/assets/schemas/variables/umm-var-form.json

```diff
@@ -583,5 +603,47 @@
         }
       ]
     }
-  ]
+  ],
+  "definitions": {
```

Comment:
what was the reason to create this definitions section and define the two fields here?

Reply:
At some point while trying to understand how to make the modifications I wanted, separating them worked better.  I have updated the schema to not include the definitions section now.